### ISSUE TITLE
finetune resnet50

### ISSUE THREAD

**Issue by txdat** [experiencor/keras-yolo2#217]
i used slim resnet50 for my own yolo2. with same code, vgg16 work well, but resnet50 bbox_loss comes to nan after some epochs, i tried to decrease learning rate from 1e-3 to 1e-6 but this still happened. can you tell me some ways to finetune resnet? thanks!

**Comment by experiencor** [comment]
@txdat I think the reason is that you don't have the resnet backend weight. I will check if I still keep it.

**Comment by txdat** [comment]
i had numpy runtimewarning about nan in compute overlaps btw pred and groundtruth box. i checked overlaps and images' annotation, no problem found. i dont understand why loss decreased normally and became nan suddenly after some epochs

**Comment by experiencor** [comment]
@txdat I don't keep the resnet backend. Are you training on your own data or COCO? You may want to use tf.Print to debug.

**Comment by txdat** [comment]
i trained on detrac dataset with imbalance data. i used weight for class=log(all_exams/class_exams) in softmax cross entropy. did you train resnet with any tricks?

**Comment by txdat** [comment]
with new your commit, it seems that yolo3 dont need warmup?

**Comment by experiencor** [comment]
I did not use any tricks. You should be able to train ResNet on COCO out of the box.

I have yet to implement new loss function of YOLOv3 but you are right. I don't see the warmup code in the darknet C++ code.
